tests/test_commands.py

```diff
@@ -478,6 +488,10 @@ def verify_reporting_row(row, time_date):
         assert row.tb_stored_now == round(
             sum(proj.size for proj in Project.query) / 1000000000000, 2
         )
+        assert (
+            row.tb_uploaded_since_start * 1e12
+            == Version.query.filter_by(id=version_id).first().size_stored
+        )
```

Comment:
As a reply to your comment in the PR - yeah, we don't have `version_id` here and this just checks uploaded since start for a specific version. My suggestion is to change it to this, which is using the same code as you currently have in the command.
```suggestion
        assert row.tb_uploaded_since_start == round(
            sum(version.size_stored for version in dds_web.utils.page_query(Version.query))
            / 1000000000000,
            2,
        )
```

After this I also suggest to change the command calculation, I've added a suggestion there too.